src/components/pages/newsletter/issues/issues.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-param-reassign */
+import group from 'core-js-pure/actual/array/group';
+import { graphql, useStaticQuery } from 'gatsby';
+import React from 'react';
+
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+import Link from 'components/shared/link';
+import CalendarIcon from 'icons/calendar.inline.svg';
+
+const Issues = () => {
+  const data = useStaticQuery(graphql`
+    query hubspotEmailsQuery {
+      hubspotEmails {
+        objects {
+          name
+          publishDate
+          isPublished
+          publishedUrl
+        }
+      }
+    }
+  `);
+
+  const getYear = (date) => {
+    const convertedDate = new Date(date);
+    const year = convertedDate.getFullYear();
+    return year;
+  };
+
+  const getMonthAndDay = (date) => {
+    const convertedDate = new Date(date);
+    const day = convertedDate.getUTCDate();
+    const month = convertedDate.toLocaleString('default', { month: 'long' });
+    return `${month}, ${day}`;
+  };
+
+  // TODO: check out if there is any way to do the filtering on the Gatsby GraphQL level
+  // AFAIK there is no way to apply filters to children nodes
+  // https://github.com/gatsbyjs/gatsby/discussions/14060
+  const items = data.hubspotEmails.objects
+    .filter((item) => item.name.match(/^eCHO news \d{1,3}$/) && item.isPublished)
+    .sort((a, b) => b.publishDate - a.publishDate);
+
+  items.forEach((item) => {
+    item.year = getYear(item.publishDate);
+
+    item.date = getMonthAndDay(item.publishDate);
+  });
+
+  // I have used a polyfill from core-js here, as the Array.Prototype.group() is only an experimental feature at the moment
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/group
+  const newsletterData = group(items, (item) => item.year);
```

Comment:
I believe we can get issues by year with JS another way, so no need to get a lib for this. 
```jsx
 const getIssues = () =>
    items.reduce((acc, item) => {
      if (!acc[item.year]) {
        acc[item.year] = [];
      } else {
        acc[item.year].push(item);
      }
      return acc;
    }, {});
```